react/features/base/lastn/middleware.js

```diff
@@ -68,15 +69,15 @@ const _updateLastN = debounce(({ dispatch, getState }) => {
     if (typeof appState !== 'undefined' && appState !== 'active') {
         lastNSelected = isLocalVideoTrackDesktop(state) ? 1 : 0;
     } else if (audioOnly) {
-        const { remoteScreenShares, tileViewEnabled } = state['features/video-layout'];
+        const { remoteScreenShares, tileViewEnabled, carMode } = state['features/video-layout'];
         const largeVideoParticipantId = state['features/large-video'].participantId;
         const largeVideoParticipant
             = largeVideoParticipantId ? getParticipantById(state, largeVideoParticipantId) : undefined;
 
         // Use tileViewEnabled state from redux here instead of determining if client should be in tile
         // view since we make an exception only for screenshare when in audio-only mode. If the user unpins
         // the screenshare, lastN will be set to 0 here. It will be set to 1 if screenshare has been auto pinned.
-        if (!tileViewEnabled && largeVideoParticipant && !largeVideoParticipant.local) {
+        if (!carMode && !tileViewEnabled && largeVideoParticipant && !largeVideoParticipant.local) {
```

Comment:
I think it's simpler if we have an if before this one, at the same level as audio-only, which checks if car mode is enabled and sets last N to 0 explicitly.

Reply:
hope I understood correctly.